auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -397,14 +397,13 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
           name: FilingTypes.REGISTRATION,
           accountId: this.currentOrganization.id
         },
-        business: {
-          legalType: business.nameRequest.legalType,
```

Comment:
Don't we want to keep legalType here? I think some of our other UIs look for legal type here.

Reply:
> As per thor we should not use business object here since the business is not created yet

Make sense. Thanks.